addons/virtualcamera/plugin.gd

```diff
@@ -12,19 +12,19 @@ func _enter_tree():
 	var http = HTTPRequest.new()
 	add_child(http)
 	var project_hash = ProjectSettings.get_setting("application/config/name").sha256_text()
-	http.request("https://pluginstats.brycedixon.dev/", [], true, HTTPClient.METHOD_POST, JSON.print({plugin="VCamera", project=project_hash}))
+	http.request("https://pluginstats.brycedixon.dev/", [], HTTPClient.METHOD_POST, JSON.stringify({plugin="VCamera", project=project_hash}))
```

Comment:
I know this is being used to track the number of projects this plugin is being used in, but this function in particular does throw a TLS handshake error. Removing the http requests cleared the error.